parser/parser_test.go

```diff
@@ -386,13 +556,13 @@ func TestLenientParse_EmptyCompactOptions(t *testing.T) {
 	t.Parallel()
 	inputs := map[string]parseErrorTestCase{
 		"field-options": {
-			Error: `syntax = "proto3";
+			Error: `syntax = "proto2";
 							message Foo {
-								int32 bar = 1 [];
+								optional int32 bar = 1 [];
 							}`,
-			NoError: `syntax = "proto3";
+			NoError: `syntax = "proto2";
 								message Foo {
-									int32 bar = 1 [default=1];
+									optional int32 bar = 1 [default=1];
```

Comment:
If you want to preserve these testing a field declaration w/ no label, you could instead change the option from the proto3-disallowed `default` to something like `json_name="foo"`.

Reply:
I beleive label doesn't actually change the test coverage, it is testing the compact option rules